flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/source/KafkaSourceLegacyITCase.java

```diff
@@ -37,7 +37,7 @@ public KafkaSourceLegacyITCase() throws Exception {
         super(true);
     }
 
-    @BeforeClass
+    @BeforeAll
     public static void prepare() throws Exception {
```

Comment:
```suggestion
    static void prepare() throws Exception {
```
could be package private

Reply:
this method calls  KafkaProducerTestBase.prepare();
Which is needs to be protected method

Reply:
ok let's change it to protected